src/main/kotlin/com/fwdekker/randomness/word/WordSettingsComponent.kt

```diff
@@ -158,16 +158,14 @@ class WordSettingsComponent(settings: WordSettings = default) : SettingsComponen
                 ValidationInfo(
                     "The longest word in the selected dictionaries is $maxWordLength characters. " +
                         "Set the minimum length to a value less than or equal to $maxWordLength.",
-                    minLength,
-                    Runnable { minLength.value = maxWordLength }
-                )
+                    minLength
+                ) { minLength.value = maxWordLength }
             maxLength.value < minWordLength ->
                 ValidationInfo(
                     "The shortest word in the selected dictionaries is $minWordLength characters. " +
                         "Set the maximum length to a value less than or equal to $minWordLength.",
-                    maxLength,
-                    Runnable { maxLength.value = minWordLength }
-                )
+                    maxLength
+                ) { maxLength.value = minWordLength }
```

Comment:
I should be able to apply quick fixes for some other issues as well, such as "Select at least one dictionary", which then selects the first dictionary in the list (if there is one).

Reply:
Quick fixes are not currently supported in the settings dialog, so I won't bother.